ambry-replication/src/main/java/com/github/ambry/replication/BackupCheckerThread.java

```diff
@@ -71,6 +84,11 @@ public BackupCheckerThread(String threadName, FindTokenHelper findTokenHelper, C
         replicationConfig, replicationMetrics, notification, storeKeyConverter, transformer, metricRegistry,
         replicatingOverSsl, datacenterName, responseHandler, time, replicaSyncUpManager, skipPredicate,
         leaderBasedReplicationAdmin);
+    try {
+      fileManager = Utils.getObj(replicationConfig.backupCheckFileManagerType, replicationConfig, metricRegistry);
+    } catch (ReflectiveOperationException e) {
+      throw new RuntimeException(e);
```

Comment:
Maybe add a log message here to make it easy to debug?

Reply:
done